config/game_options.txt

```diff
@@ -512,7 +512,8 @@ MICE_ROUNDSTART 10
 ## Determines if players are allowed to print integrated circuits, uncomment to allow.
 #IC_PRINTING
 
-## Uncomment to allow roundstart trait selection in the character setup menu.
+## Uncomment to allow roundstart quirk selection in the character setup menu.
+## This used to be named traits, hence the config name, but it handles quirks, not the other kind of trait!
 ROUNDSTART_TRAITS
```

Comment:
You should rename this here, there's a way to make configs map to other configs for legacy purposes. (I think the FPS config does it)